### ISSUE TITLE
Serial number wrong on Upnp description.xml

### ISSUE THREAD

**Issue by johnzielke** [dresden-elektronik/deconz-rest-plugin#948]
When calling GET /description.xml on my deconz brige, the xml looks like this:
```
<root xmlns="urn:schemas-upnp-org:device-1-0">
  <script/>
  <specVersion>
    <major>1</major>
    <minor>0</minor>
  </specVersion>
  <URLBase>http://192.168.178.49:80/</URLBase>
  <device>
    <deviceType>urn:schemas-upnp-org:device:Basic:1</deviceType>
    <friendlyName>
      Philips hue (192.168.178.49) compatible Wireless Light Control Gateway
    </friendlyName>
    <manufacturer>Royal Philips Electronics</manufacturer>
    <manufacturerURL>http://www.dresden-elektronik.de</manufacturerURL>
    <modelDescription>dresden elektronik Wireless Light Control</modelDescription>
    <modelName>Philips hue bridge 2012</modelName>
    <modelNumber>929000226503</modelNumber>
    <modelURL>http://www.dresden-elektronik.de</modelURL>
    <serialNumber>{{BRIDGEID}}</serialNumber>
    <UDN>uuid:e88e1818-d762-4657-b7b3-553283511765</UDN>
    <gatewayName>Phoscon-GW</gatewayName>
    <presentationURL>index.html</presentationURL>
    <iconList>
      <icon>
        <mimetype>image/png</mimetype>
        <height>48</height>
        <width>48</width>
        <depth>24</depth>
        <url>hue_logo_0.png</url>
      </icon>
      <icon>
        <mimetype>image/png</mimetype>
        <height>120</height>
        <width>120</width>
        <depth>24</depth>
        <url>hue_logo_3.png</url>
      </icon>
    </iconList>
  </device>
</root>
```

Note that it says <serialNumber>**{{BRIDGEID}}**</serialNumber>, which causes errors in tools like Openhab when trying to do a discovery. Is this a bug in the software, or some misconfiguration on my side? (I'd look into it, and try to fix it myself, but I don't deal with c++ too much, and am currently expanding the openhab hue binding and got enough to do there)

**Comment by manup** [comment]
Which deCONZ version do you use?

**Comment by johnzielke** [comment]
This is my config, when calling the API, the software version is there:
```
"config": {
        "UTC": "2018-11-14T22:59:48",
        "apiversion": "1.0.9",
        "backup": {
            "errorcode": 0,
            "status": "idle"
        },
        "bridgeid": "00212E****",
        "datastoreversion": "60",
        "devicename": "ConBee",
        "dhcp": true,
        "factorynew": false,
        "fwversion": "0x26240500",
        "gateway": "192.168.178.1",
        "internetservices": {
            "remoteaccess": "disconnected"
        },
        "ipaddress": "192.168.***",
        "linkbutton": false,
        "localtime": "2018-11-14T23:59:48",
        "mac": "***:8d",
        "modelid": "deCONZ",
        "name": "Phoscon-GW",
        "netmask": "255.255.255.0",
        "networkopenduration": 180,
        "panid": 38616,
        "portalconnection": "disconnected",
        "portalservices": false,
        "portalstate": {
            "communication": "disconnected",
            "incoming": false,
            "outgoing": false,
            "signedon": false
        },
        "proxyaddress": "none",
        "proxyport": 0,
        "replacesbridgeid": null,
        "rfconnected": true,
        "starterkitid": "",
        "swupdate": {
            "checkforupdate": false,
            "devicetypes": {
                "bridge": false,
                "lights": [],
                "sensors": []
            },
            "notify": false,
            "text": "",
            "updatestate": 0,
            "url": ""
        },
        "swupdate2": {
            "autoinstall": {
                "on": false,
                "updatetime": ""
            },
            "bridge": {
                "lastinstall": "",
                "state": "allreadytoinstall"
            },
            "checkforupdate": false,
            "install": false,
            "lastchange": "",
            "lastinstall": "",
            "state": "allreadytoinstall"
        },
        "swversion": "2.5.40",
        "timeformat": "24h",
        "timezone": "Etc/GMT-1",
        "uuid": "****3511765",
        "websocketnotifyall": true,
        "websocketport": 443,
        "whitelist": ***
        "zigbeechannel": 15
}
```

**Comment by KodeCR** [comment]
Yes, this was fixed in 2.05.44, see #902.
